models/place-post-deprecated.json

```diff
@@ -0,0 +1,92 @@
+{
+  "title": "place-post-deprecated",
+  "type": "object",
+  "x-internal": true,
+  "properties": {
+    "mainLanguage": {
+      "$ref": "./place-mainLanguage.json"
+    },
+    "name": {
+      "type": "string",
+      "pattern": "\\S",
+      "minLength": 1,
+      "description": "The name of the place.",
+      "example": "M – Museum"
+    },
+    "type": {
+      "type": "object",
+      "required": [
+        "id",
+        "label"
+      ],
+      "description": "The term (type) used to categorize the place. Terms are pre-defined and can be browsed in our [Terms](../docs/terms.md) guide.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "example": "GnPFp9uvOUyqhOckIFMKmg",
+          "description": "Unique id of the term (type)."
+        },
+        "label": {
+          "type": "string",
+          "example": "Museum of galerij",
+          "description": "Human-readable label of the term (type). Currently only available in Dutch."
+        },
+        "domain": {
+          "type": "string",
+          "example": "eventtype",
+          "description": "The domain of the term (type)."
+        }
```

Comment:
This should be an enum with `eventtype` and `facility`

Reply:
Does a facility makes sense when creating a new place?

Reply:
Ah I was confused. This should indeed only allow `eventtype` terms. So an enum of just `eventtype` then?

Reply:
But I don't think this is used for actual validation so more of a nit

Reply:
I will just use `eventtype`